code/src/components/thoughts-list.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&display=swap');
+.thoughts-list-container {
+  font-family: 'Inconsolata', monospace;
+  display: flex;
+  justify-content: center;
+}
+
+.thought-container {
+  border: 2px solid #000000;
+  background-color: #FFFFFF;
```

Comment:
You can shorten the HEX color code for white to `#fff` and black to `#000`. Less to write 😉